src/sass/app_2/_common/themes/dark.scss

```diff
@@ -108,10 +108,37 @@
             }
         }
     }
-    .btn-purchase:disabled {
-        box-shadow: 0 2px 2px 0 $COLOR_DARK_BLUE_7;
-        background: $COLOR_DARK_BLUE_6;
-        color: $COLOR_DARK_BLUE_8;
+    .btn-purchase {
+        &:disabled {
+            box-shadow: 0 2px 2px 0 $COLOR_LIGHT_BLACK_3;
+            background-color: $COLOR_DARK_BLUE_8;
+            /* stylelint-disable */
```

Comment:
what rule is failing in stylelint? better to disable that rather than all of stylelint